migrations/1532514271014_add-additional-tracked-actions.js

```diff
@@ -0,0 +1,14 @@
+exports.up = (pgm) => {
+  pgm.db.query(
+    `INSERT INTO action(code, description) VALUES
+    ('InstalledZapp', 'User installed Zapp.'),
+    ('UninstalledZapp', 'User uninstalled Zapp.'),
+    ('HeatingFirstLoginDone', 'User clicked "Done!" on first login reminder.'),
+    ('HeatingFirstLoginNotNow', 'User clicked "Not now" on first login reminder.');`
+  )
+}
+
+exports.down = (pgm) => {
+  pgm.db.query(`DELETE FROM action_log;`)
+  pgm.db.query(`DELETE FROM action;`)
```

Comment:
This isn't really the correct down, since it would also delete the other actions. For correctness, this should really have a load of `WHERE code = 'InstalledZapp' OR code = '…'`, etc. type lines.